dracut-module-setup.sh

```diff
@@ -973,21 +973,24 @@ kdump_install_random_seed() {
 
 kdump_install_systemd_conf() {
     # Kdump turns out to require longer default systemd mount timeout
-    # than 1st kernel(90s by default), we use default 300s for kdump.
-    if ! grep -q -r "^[[:space:]]*DefaultTimeoutStartSec=" "${initdir}/etc/systemd/system.conf"*; then
-        mkdir -p "${initdir}/etc/systemd/system.conf.d"
-        echo "[Manager]" > "${initdir}/etc/systemd/system.conf.d/kdump.conf"
-        echo "DefaultTimeoutStartSec=300s" >> "${initdir}/etc/systemd/system.conf.d/kdump.conf"
-    fi
+    # than 1st kernel(45s by default), we use default 300s for kdump.
+    mkdir -p "${initdir}/etc/systemd/system.conf.d"
+    cat > "${initdir}/etc/systemd/system.conf.d/99-kdump.conf" << EOF
+[Manager]
+DefaultTimeoutStartSec=300s
```

Comment:
Hi Coiby,

in theory users could need a value larger than 300s. But in practice I would say that 300s (5 minutes) should be sufficient.

But that's the point I wanted to make. Currently there is only the check whether the user has DefaultTimeoutStartSec set. But there is no check whether it's larger than our suggested 300s or not. So it could also be much smaller. What I'm not entirely sure is how we should handle such situations. Especially as I'm not sure if we can simply take over the config from the production system to the kdump initrd as the user might dump to a target that is usually not mounted in production...

All in all adding the config unconditionally is the most simple and consistent solution in my opinion. But as noted in  the commit message I don't oppose to keep the check and/or extend it. That's what I wanted to discuss here.

Reply:
Since 5 minutes is sufficient, the patch looks good to me! Thanks for the clarification!